bootstrap_cfn/config.py

```diff
@@ -225,6 +227,17 @@ def elb(self):
                     listener.update(ssl_template)
 
 
+            elb_sg = template.pop('DefaultELBSecurityGroup')
+            if 'security_groups' in elb:
+                for sg_name, sg in elb['security_groups'].items():
+                    new_sg = deepcopy(elb_sg)
+                    new_sg['Properties']['SecurityGroupIngress'] = sg
+                    elb_sgs[sg_name + safe_name] = new_sg
```

Comment:
Security group names can contain "a-z, A-Z, 0-9, spaces, and ._-:/()#,@[]+=&;{}!$*" so we should be able to use the raw SG name without modification - this makes it clearer to parse the SG name - currently we would end up with something like "DefaultSGtesttemplateext" but "DefaultSG-test-template-ext" would be much preferable.